library/ui-styles/src/main/res/values/styles_buttons.xml

```diff
@@ -15,7 +15,7 @@
         <item name="android:textColor">@android:color/white</item>
     </style>
 
-    <style name="Widget.Vector.Button.Destructive">
+    <style name="Widget.Vector.Button.Destructive" parent="BottomSheetButtonTextColor">
```

Comment:
By doing that you delete the previous parent relation to `Widget.Vector.Button`. This is not good.

Also the naming `BottomSheetButtonTextColor` is a bit weird. I would delete this new style, and update the style `VectorMaterialThemeOverlayDestructive` (and maybe `VectorMaterialThemeOverlayPositive`) to 

<img width="488" alt="image" src="https://user-images.githubusercontent.com/3940906/154505630-1ab18a41-1bc1-4945-8ad1-24d5c8867c91.png">

WDYT?